components/atom/textarea/src/index.js

```diff
@@ -8,8 +8,23 @@ const SIZES = {
   LONG: 'long'
 }
 
-const AtomTextarea = ({onChange, size, value, ...props}) => {
-  const className = cx(BASE_CLASS, `${BASE_CLASS}--${size}`)
+const ERROR_STATES = {
+  ERROR: 'error',
+  SUCCESS: 'success'
+}
+
+const getErrorStateClass = errorState => {
```

Comment:
It might be easier to use `classNames` library to handle different class names based on some values

Reply:
it is using className in line 23..

Reply:
refactored for all components: molecule/textarea, molecule/autosuggest, atom/input, molecule/select